hack/verify-vendor.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+WMCO_ROOT=$(pwd)
+
+# create a nice clean place to put our new vendor tree
+mkdir -p "${WMCO_ROOT}/_tmp"
+_tmpdir="$(mktemp -d "${WMCO_ROOT}/_tmp/wmco-vendor.XXXXXX")"
+
+# Copy the contents of the kube directory into the nice clean place (which is NOT shaped like a GOPATH)
```

Comment:
`shaped like a GOPATH`? What is the shape of a GOPATH? :-D

Reply:
like `/home/ravig/go` and having src and bin etc, if there it would be treated like a GOPATH. However I can remove it.

Reply:
You are talking about the directory layout. I am fine if you rephrase with that terminology.